dashboard/test/empty_test.rb

```diff
@@ -0,0 +1,19 @@
+$start_time = Time.now
```

Comment:
Why the test is called empty_test? What is this test testing?

Reply:
It's from another task 😅 removed